deployments/auth.yaml

```diff
@@ -35,7 +35,13 @@ rules:
 - apiGroups:
   - ""
   - k8s.cni.cncf.io
+  - extensions
+  - apps
   resources:
```

Comment:
do we need to include deployments in this list as well?

Reply:
nvm, just realized that replicaset will be the owner

Reply:
The replicaset is the ownerref of pod when its deployed with Deployment kind, The replicaset's ownerref is Deployment. 
Hence this check would not be needed.